raiden/utils/upgrades.py

```diff
@@ -17,12 +17,13 @@
 from raiden.storage.versions import VERSION_RE, older_db_file
 from raiden.utils.typing import Callable, Optional
 
-UPGRADES_LIST = [
-    upgrade_v16_to_v17,
-    upgrade_v17_to_v18,
-    upgrade_v18_to_v19,
-    upgrade_v19_to_v20,
-]
+# Source Target -> Upgrade Func
+UPGRADES_LIST = {
```

Comment:
please rename to UPGRADES_MAP as it's now a mapping

Also check my comment below for why I don't think this should be a mapping